json.ml

```diff
@@ -187,14 +193,16 @@ let print_json_state client_id procs handlers aps =
 module JsonState = struct
   type t = {
     client_id : client_id;
+    ws_conn_url_opt : maybe_connection_url;
```

Comment:
(2/2) ... [ continuation of the above ] then `ws_conn_url_opt` could have type `connection_url option` this way it is the possible constructors are immediate from the interface. (also `ws_conn_url_opt` could be renamed `ws_conn_url` as `opt` is redundant in either case)